src/core/HyperlanePermissionlessDeployer.ts

```diff
@@ -100,54 +109,81 @@ export class HyperlanePermissionlessDeployer {
     );
   }
 
-  get chains(): ChainName[] {
+  get skipLocalDeploy(): boolean {
+    return !this.isDeployChain(this.local);
+  }
+
+  get remoteDeployChains(): ChainName[] {
+    return this.remotes.filter((chain) => this.isDeployChain(chain));
+  }
+
+  get deployChains(): ChainName[] {
```

Comment:
Would `deployableChains` be a slightly more accurate name throughout this file than `deployChains`?